src/stats/stats-service-openmetrics.c

```diff
@@ -141,8 +141,8 @@ openmetrics_export_metric_value(struct openmetrics_request *req, string_t *out,
 			    timestamp);
 		break;
 	case OPENMETRICS_METRIC_TYPE_DURATION:
-		str_printfa(out, " %"PRIu64" %"PRId64"\n",
-			    stats_dist_get_sum(metric->duration_stats),
+		str_printfa(out, " %f %"PRId64"\n",
```

Comment:
Maybe %.3f?

(Otherwise I'll leave this to others, since I haven't really looked at openmetrics myself.)

Reply:
Thanks, makes sense, since we only have ms precision.

Reply:
Applied a change to .6f